node/src/index_node.ts

```diff
@@ -3,3 +3,7 @@ export { sslConfiguration } from './server_certificates'
 export * as httpHttpsServer from './http_https_server'
 export { setHttpsAndServerForKarma } from './karma_https_config'
 export { Arguments as BrowserFlags } from './arguments'
+export { BrowserStackSessionFactory, BrowserStackSessionFactoryConfig } from './browserstack_session_factory'
+export { CapabilitiesFactory } from './capabilities_factory'
+export { getBrowserStackCredentials } from './browserstack_helpers'
+export { makeBrowserStackLocalManagerFactory } from './browserstack_local_manager'
```

Comment:
This function exists only because of the Karma dependency injector. Let's export the class instead, like for the other services.

Reply:
same for the BrowserStackCredentials?

Reply:
No, there is no such class